packages/gatsby-remark-prismjs/src/__tests__/load-prism-language.js

```diff
@@ -0,0 +1,25 @@
+const loadPrismLanguage = require(`../load-prism-language`)
+
+describe(`load prism language`, () => {
+  it(`throw if language not support`, () => {
+    expect(() => loadPrismLanguage(`imnotalanguage`)).toThrow(
+      `Prism doesn't support language 'imnotalanguage'.`
+    )
+  })
+
+  it(`also load the required language`, () => {
+    const language = `cpp`
+    const requiredLanguage = `c`
+    const Prism = require(`prismjs`)
```

Comment:
I'd like to isolate the `prismjs` module for each test so that tests won't affect each other. But it seems no easy way. any ideas? @KyleAMathews

Reply:
Eh, dunno. Not enough of a Jest expert to know how to do this.

Reply:
Ooo… but now I am :-) https://facebook.github.io/jest/docs/en/configuration.html#resetmodules-boolean

Reply:
`jest.resetModules()`

Reply:
@KyleAMathews Works great! and it's ready to review now.